pep-0701.rst

```diff
@@ -320,7 +340,75 @@ All restrictions mentioned in the PEP are lifted from f-string literals, as expl
   expanded when the innermost string is evaluated.
 * Comments, using the ``#`` character, are possible only in multi-line f-string literals,
   since comments are terminated by the end of the line (which makes closing a
-  single-line f-string literal impossible)
+  single-line f-string literal impossible). Comments in multi-line f-string literals require
+  the closing ``{`` of the expression part to be present in a different line as the one the
+  comment is in.
+
+.. _considerations_of_quote_reuse:
+
+Considerations regarding quote reuse
+------------------------------------
+
+One of the consequences of the grammar proposed here is that, as mentioned above,
+f-string expressions can now contain strings delimited with the same kind of quote
+that is used to delimit the external f-string literal. For example:
+
+    >>> f" something { my_dict["key"] } something else "
+
+In the `discussion thread of this document <https://discuss.python.org/t/pep-701-syntactic-formalization-of-f-strings>`_
```

Comment:
```suggestion
In the `discussion thread for this PEP <https://discuss.python.org/t/pep-701-syntactic-formalization-of-f-strings/22046>`_,
```

* Fix grammar
* Use more standard nomenclature
* Include thread ID in URL to be more reliable
* Add missing comma